examples/ConfigOnDoubleReset/ConfigOnDoubleReset.ino

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
Yes, a nice contribution from @datacute. I updated the documentation to reflect this change. Looking at some of the other @tzapu   comments.

> -most of my sensors do not have a button

I can't think of any other way to get human initiation which I think is necessary and the @datacute solution is good if the sensors have a reset button. 

> -some of those sensors are running on battery, adding a 60 second delay on every wake up, would be a killer of battery life

Yes I don't like that one either. I see someone else made a non blocking configuration portal but I'm not keen on that because it changes timing within the user program which is often problematic for IOT applications. 

> -a lot of the issues as you say, is mostly calling WiFi.begin(), why change the whole flow, remove autoConnect(), etc ? 

I set out the reasons for [deprecating  autoConnect()](https://github.com/kentaylor/WiFiManager#configuration-portal-initiation-not-automatic) . I find that it works well during development but when deployed it is rare that new credentials are required when there is a loss of WiFi network visibility.

> -id like to mention this again, my goal is to keep this as lightweight and as simple as possible, somebody that picks it up for the first time, together with picking up arduino ide for the first time, should be able to use it in 30 seconds. power users will be able to modify and suit their needs anyway.

Yes. The [autoConnect example](https://github.com/tzapu/WiFiManager/blob/master/examples/AutoConnect/AutoConnect.ino) is wonderfully simple. I wonder if autoconnect could be redefined to mean start up a configuration portal if no SSID has been defined. That would avoid getting a configuration whenever the WiFi network fails and retains the simplicity.